omnigibson/systems/micro_particle_system.py

```diff
@@ -1440,10 +1440,7 @@ def _create_particle_prototypes(self):
         prototype = lazy.pxr.UsdGeom.Sphere.Define(og.sim.stage, prototype_prim_path)
         prototype.CreateRadiusAttr().Set(self.particle_radius)
         relative_prototype_prim_path = absolute_prim_path_to_scene_relative(self._scene, prototype_prim_path)
-        load_config = {"created_manually": True}
```

Comment:
this is good! seems hacky. so is there ever a reason why we would need `create_manually`?

Reply:
see my comments above